### PR TITLE
Update graph-bamboohr.md

### PR DESCRIPTION
**Pull request by janettelynch**

### CODE REVIEW

File: knowledgeBase/APIs_and-integrations/HRMS/graph-bamboohr.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Integration with JupiterOne
+# BambooHR with JupiterOne
```
Comment (austinkelleher) [inline knowledgeBase/APIs_and-integrations/HRMS/graph-bamboohr.md:1]: @janettelynch Will this line of code get re-written the next time that we sync the integration files?

Comment (janettelynch) [reply knowledgeBase/APIs_and-integrations/HRMS/graph-bamboohr.md:1]: @austinkelleher Maybe? Bryan had changed all the code to put the names of the integrations in the H1 of each article. I'll check the Bamboo code.

**Comment by janettelynch** [comment]
I checked https://github.com/JupiterOne/docs/pull/229 which had the integration name taken out of the H1. But, when I look at prod the names are still in the H1. I'll check on this change after it is merged to prod.

**Comment by janettelynch** [comment]
@austinkelleher Can you approve so I can merge to staging. Thanks!

**Review by ninetreats — APPROVED** [review]
Looks good.

**Pull request merged**